Gems/ROS2/Code/Source/ROS2ModuleInterface.h

```diff
@@ -61,7 +61,6 @@ namespace ROS2
                   AckermannControlComponent::CreateDescriptor(),
                   RigidBodyTwistControlComponent::CreateDescriptor(),
                   SkidSteeringControlComponent::CreateDescriptor(),
-                  ROS2CameraSensorComponent::CreateDescriptor(),
```

Comment:
FYI for other reviewers, this is just removing a repetition of `ROS2CameraSensorComponent::CreateDescriptor(),`, this runtime component's descriptor is already included in line 60.

Reply:
Thanks!